test/basic.jl

```diff
@@ -109,6 +109,9 @@ end
     k2 = spacekernel(kfft, axes(k); rfftsz=31)
     @test k2 ≈ k
 
+    k = centered([0, 1, 0] * [0, 1, 0]') # Kronecker impulse
+    @test freqkernel(k, (8, 8)) ≈ ones(8,8) # flat frequency response
```

Comment:
Would be good to test your exciting new size-checking code and make sure it returns the correct error messages.

Reply:
done